operator/pkg/model/translation/gateway-api/translator.go

```diff
@@ -63,12 +63,14 @@ func (t *translator) Translate(m *model.Model) (*ciliumv2.CiliumEnvoyConfig, *co
 	}
 
 	// Set the owner reference to the CEC object.
+	isController := true
 	cec.OwnerReferences = []metav1.OwnerReference{
 		{
 			APIVersion: gatewayv1beta1.GroupVersion.String(),
 			Kind:       source.Kind,
 			Name:       source.Name,
 			UID:        types.UID(source.UID),
+			Controller: &isController,
```

Comment:
nit: there is a small utility for getting addr

```suggestion
			Controller: model.AddressOf(true),
```

Reply:
ah nice, was looking for this. wanted to avoid adding an additional utility :) will fix this

Reply:
changed to use `model.AddressOf(...)`